deepchem/models/losses.py

```diff
@@ -25,24 +22,38 @@ def __call__(self, output, labels):
     """
     raise NotImplementedError("Subclasses must implement this")
 
+  def _create_pytorch_loss(self):
```

Comment:
Losses work differently in TensorFlow and PyTorch.  For TensorFlow, `_compute_tf_loss()` directly computes the loss function, so it takes outputs and labels.  For PyTorch, it creates and returns an object that computes the loss function.  `_create_pytorch_loss()` doesn't take any arguments because it doesn't need them.  The object it returns is callable and takes outputs and labels as arguments.